2023-04-04 19:43:40,481 INFO    MainThread:24832 [wandb_setup.py:_flush():76] Configure stats pid to 24832
2023-04-04 19:43:40,481 INFO    MainThread:24832 [wandb_setup.py:_flush():76] Loading settings from C:\Users\PAVILION 15-DW001LA\.config\wandb\settings
2023-04-04 19:43:40,481 INFO    MainThread:24832 [wandb_setup.py:_flush():76] Loading settings from C:\Users\PAVILION 15-DW001LA\Documents\Tesis\Códigos\RNAs\Red_Densa\wandb\settings
2023-04-04 19:43:40,481 INFO    MainThread:24832 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-04-04 19:43:40,481 INFO    MainThread:24832 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-04-04 19:43:40,481 INFO    MainThread:24832 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'RNAs\\Red_Densa\\RNA_densa.py', 'program': 'C:\\Users\\PAVILION 15-DW001LA\\Documents\\Tesis\\Códigos\\RNAs\\Red_Densa\\RNA_densa.py'}
2023-04-04 19:43:40,482 INFO    MainThread:24832 [wandb_init.py:_log_setup():506] Logging user logs to C:\Users\PAVILION 15-DW001LA\Documents\Tesis\Códigos\RNAs\Red_Densa\wandb\run-20230404_194340-5olliy14\logs\debug.log
2023-04-04 19:43:40,482 INFO    MainThread:24832 [wandb_init.py:_log_setup():507] Logging internal logs to C:\Users\PAVILION 15-DW001LA\Documents\Tesis\Códigos\RNAs\Red_Densa\wandb\run-20230404_194340-5olliy14\logs\debug-internal.log
2023-04-04 19:43:40,482 INFO    MainThread:24832 [wandb_init.py:init():546] calling init triggers
2023-04-04 19:43:40,482 INFO    MainThread:24832 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {}
2023-04-04 19:43:40,482 INFO    MainThread:24832 [wandb_init.py:init():602] starting backend
2023-04-04 19:43:40,482 INFO    MainThread:24832 [wandb_init.py:init():606] setting up manager
2023-04-04 19:43:40,486 INFO    MainThread:24832 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-04-04 19:43:40,494 INFO    MainThread:24832 [wandb_init.py:init():613] backend started and connected
2023-04-04 19:43:40,497 INFO    MainThread:24832 [wandb_init.py:init():701] updated telemetry
2023-04-04 19:43:40,550 INFO    MainThread:24832 [wandb_init.py:init():741] communicating run to backend with 60.0 second timeout
2023-04-04 19:43:40,953 INFO    MainThread:24832 [wandb_run.py:_on_init():2133] communicating current version
2023-04-04 19:43:41,163 INFO    MainThread:24832 [wandb_run.py:_on_init():2142] got version response 
2023-04-04 19:43:41,163 INFO    MainThread:24832 [wandb_init.py:init():789] starting run threads in backend
2023-04-04 19:43:45,694 INFO    MainThread:24832 [wandb_run.py:_console_start():2114] atexit reg
2023-04-04 19:43:45,694 INFO    MainThread:24832 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2023-04-04 19:43:45,695 INFO    MainThread:24832 [wandb_run.py:_redirect():2034] Wrapping output streams.
2023-04-04 19:43:45,695 INFO    MainThread:24832 [wandb_run.py:_redirect():2059] Redirects installed.
2023-04-04 19:43:45,695 INFO    MainThread:24832 [wandb_init.py:init():831] run started, returning control to user process
2023-04-04 19:43:45,696 INFO    MainThread:24832 [wandb_config.py:__setitem__():152] config set learning_rate = 2.5e-05 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001F4318977C0>>
2023-04-04 19:43:45,696 INFO    MainThread:24832 [wandb_run.py:_config_callback():1251] config_cb learning_rate 2.5e-05 None
2023-04-04 19:43:45,696 INFO    MainThread:24832 [wandb_config.py:__setitem__():152] config set batch_size = 20 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001F4318977C0>>
2023-04-04 19:43:45,696 INFO    MainThread:24832 [wandb_run.py:_config_callback():1251] config_cb batch_size 20 None
2023-04-04 19:43:45,697 INFO    MainThread:24832 [wandb_config.py:__setitem__():152] config set epochs = 50 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001F4318977C0>>
2023-04-04 19:43:45,697 INFO    MainThread:24832 [wandb_run.py:_config_callback():1251] config_cb epochs 50 None
2023-04-04 19:43:45,697 INFO    MainThread:24832 [wandb_config.py:__setitem__():152] config set metric = mape - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001F4318977C0>>
2023-04-04 19:43:45,698 INFO    MainThread:24832 [wandb_run.py:_config_callback():1251] config_cb metric mape None
2023-04-04 19:43:45,698 INFO    MainThread:24832 [wandb_config.py:__setitem__():152] config set loss = mse - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001F4318977C0>>
2023-04-04 19:43:45,698 INFO    MainThread:24832 [wandb_run.py:_config_callback():1251] config_cb loss mse None
2023-04-04 19:43:45,698 INFO    MainThread:24832 [wandb_config.py:__setitem__():152] config set optimizer = rmsprop - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001F4318977C0>>
2023-04-04 19:43:45,699 INFO    MainThread:24832 [wandb_run.py:_config_callback():1251] config_cb optimizer rmsprop None
2023-04-04 19:44:26,399 INFO    MainThread:24832 [wandb_config.py:__setitem__():152] config set predic = [0.06973326206207275, 0.03529277443885803, 0.03624764084815979, 0.07019990682601929] - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001F4318977C0>>
2023-04-04 19:44:26,399 INFO    MainThread:24832 [wandb_run.py:_config_callback():1251] config_cb predic [0.06973326206207275, 0.03529277443885803, 0.03624764084815979, 0.07019990682601929] None
2023-04-04 19:44:26,401 INFO    MainThread:24832 [wandb_config.py:__setitem__():152] config set correct = [0.06890000000000095, 0.03495000000000049, 0.035950000000000516, 0.06990000000000098] - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001F4318977C0>>
2023-04-04 19:44:26,401 INFO    MainThread:24832 [wandb_run.py:_config_callback():1251] config_cb correct [0.06890000000000095, 0.03495000000000049, 0.035950000000000516, 0.06990000000000098] None
2023-04-04 19:45:13,364 WARNING MsgRouterThr:24832 [router.py:message_loop():77] message_loop has been closed
